### PR TITLE
Made changes to iocage docs. Came from ticket48768.

### PR DESCRIPTION
**Pull request by a-stjohn**
Make sure to follow and check these boxes before submitting a PR! Thank you.

- [ x] Explain the feature
Updates to the docs in regard to ticket 48768
- [ x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: iocage.8 [hunk 1]
```diff
@@ -1804,6 +1816,11 @@ Default: none
 .Pp
 Source:
 .Xr jail(8)
+.It Pf vnet_defualt_interface= Op none | Ar INTERFACE
+Defualt network interface used for the VNET bridge interface in the jail.
+Only takes effect when VNET is set and bridge interfaces are not active.
```
Comment (skarekrow) [inline iocage.8]: Drop the bridges are active statement here

File: iocage.8 [hunk 2]
```diff
@@ -1804,6 +1816,11 @@ Default: none
 .Pp
 Source:
 .Xr jail(8)
+.It Pf vnet_defualt_interface= Op none | Ar INTERFACE
+Defualt network interface used for the VNET bridge interface in the jail.
```
Comment (skarekrow) [inline iocage.8]: Default

File: iocage.8 [hunk 3]
```diff
@@ -1804,6 +1816,11 @@ Default: none
 .Pp
 Source:
 .Xr jail(8)
+.It Pf vnet_defualt_interface= Op none | Ar INTERFACE
```
Comment (skarekrow) [inline iocage.8]: Default

File: iocage.8 [hunk 4]
```diff
@@ -1804,6 +1816,11 @@ Default: none
...
+Defualt network interface used for the VNET bridge interface in the jail.
+Only takes effect when VNET is set and bridge interfaces are not active.
+.Pp
+Defualt: none
```
Comment (skarekrow) [inline iocage.8]: Default

File: iocage.8 [hunk 5]
```diff
@@ -1856,8 +1873,8 @@ Also see
 for how traffic is handled.
 Basically, bridges behave like a network switch.
 .Pp
-The PF firewall is not supported inside VNET jails as of July 2014.
-PF can be enabled for the host.
+The PF firewall is supported inside VNET jails, and uses VNET for
+automated testing of PF.
```
Comment (skarekrow) [inline iocage.8]: Let’s drop the whole pf statement as it isn’t relevant anymore.

File: iocage.8 [hunk 6]
```diff
@@ -1856,8 +1873,8 @@ Also see
...
 IPFW is fully supported inside a VNET jail.
```
Comment (skarekrow) [inline iocage.8]: Lets say IPFW/pf here

File: iocage.8 [hunk 7]
```diff
@@ -498,15 +498,16 @@ UUID/NAME wrapper for
...
-also runs commands similar to 
+also runs commands similar to
 .Nm .
 When using
 .Xr jexec
 use the JID instead of the jail name.
-For more info see the manual page for 
+For more info see the manual page for
 .Xr jexec .
+Use -- in front of specified options to prevent iocage from parsing them.
```
Comment (skarekrow) [inline iocage.8]: s/options/command/

**Review by skarekrow — CHANGES_REQUESTED** [review]

File: iocage.8 (hunk 6 above)
Comment (a-stjohn) [reply iocage.8]: Warren does not like slashes. So is this "and" or an "or"? (that, being the reason he doesn't like them)

Comment (skarekrow) [reply iocage.8]: Then say `IPFW and pf` :)

File: iocage.8 (hunk 7 above)
Comment (skarekrow) [reply iocage.8]: Why did you mark this resolved? This isn't just options that need it, it's the whole command they specify.
`iocage exec JAIL -- COMMAND` not just options.

File: iocage.8 [hunk 8]
```diff
@@ -1605,6 +1605,18 @@ Default: 0
 .Pp
 Source:
 .Xr jail 8
+.It Pf allow_tun= Op 0 | 1
+Unhides tun devices for the jail with a individual devfsruleset.
```
Comment (skarekrow) [inline iocage.8]: `an`

**Review by skarekrow — CHANGES_REQUESTED** [review]
You marked all these as resolved when none of them have been, was that by mistake? Regardless, one more review note.

File: iocage.8 [hunk 9]
```diff
@@ -1873,9 +1874,8 @@ Also see
 for how traffic is handled.
 Basically, bridges behave like a network switch.
 .Pp
-The PF firewall is supported inside VNET jails, and uses VNET for
-automated testing of PF.
-IPFW is fully supported inside a VNET jail.
+The PF firewall is supported inside VNET jails.
```
Comment (skarekrow) [inline iocage.8]: Drop this line still :) it's said in the very next line again.

**Review by skarekrow — CHANGES_REQUESTED** [review]

**Review by skarekrow — APPROVED** [review]

**Pull request merged**